modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java

```diff
@@ -16,8 +16,12 @@
 
 package org.apache.ignite.internal.processors.cache.persistence.pagemem;
 
+import java.nio.ByteBuffer;
```

Comment:
unused imports ? here and some lines near?

Reply:
Technically they are not unused. Imports were added for javadoc method reference.